lib/core/DocsLayout.js

```diff
@@ -61,7 +64,15 @@ class DocsLayout extends React.Component {
               {metadata.previous_id && (
                 <a
                   className="docs-prev button"
-                  href={metadata.previous_id + '.html'}>
+                  href={
+                    config.baseUrl +
+                    docsDir +
+                    '/' +
+                    language +
```

Comment:
This might cause a bug here. If you compare this with the default website the `/en/` is not part of urls. 

But with this PR `/en/` starts appearing in urls:

![image](https://user-images.githubusercontent.com/128755/38175124-a0a0b18e-35a5-11e8-922a-e1e648e6cf3f.png)

Maybe that should be determined by `useEnglishUrl` option

Reply:
Hi, thanks for your feedback! By default website you mean [docusaurus.io](https://www.docusaurus.io)? If so, `/en/` already appears to be a part of the url (https://docusaurus.io/docs/en/installation.html).

Reply:
@filkra sorry, I meant the website you get after you run `docusaurus-init`

Reply:
This seems to be more complicated than I thought... Using only `useEnglishUrl` doesn't work since the `/en/` part will also be inserted, if a `languages.js` file exists. I ran `docusaurus-init` and created a `languages.js` file containing the following.

```js
const languages = [
  {
    enabled: true,
    name: "English",
    tag: "en"
  }
];
module.exports = languages;
```
`useEnglishUrl` is set to false and the URL still contains `/en/`. Is this the right behaviour?